src/main/java/io/meeds/qa/ui/pages/page/factory/tasks/TasksPage.java

```diff
@@ -1579,6 +1582,12 @@ public void setInSearchProjectField(String project) {
     verifyPageLoaded();
   }
 
+  public void setInSearchSpaceField(String space) {
+    searchSpaceInput.setTextValue(space);
+    waitFor(1).seconds();
```

Comment:
1. Try to check on progress bar instead of explicit waiting
2. Try to reuse existing scenario definition :
https://github.com/Meeds-io/meeds-qa-ui/blob/c01e8c4626e2a571abc351cdedaf9d51f9000622/src/test/java/io/meeds/qa/ui/steps/definition/ManageSpaceStepDefinitions.java#L344-L347
3. Avoid defining `Spaces` page step inside `Task` page step